rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2485,6 +2486,23 @@ def test_upgrade_db_31_to_32(user_data_dir):  # pylint: disable=unused-argument
             'SELECT COUNT(*) FROM sqlite_master WHERE type="table" AND name=?', (name,),
         ).fetchone()[0] == 0
 
+        db_v31.logout()
+
+    # Check that trades with fee missing sets fee_currency to NULL and vice versa
+    db = _init_db_with_target_version(
+        target_version=32,
+        user_data_dir=user_data_dir,
+        msg_aggregator=msg_aggregator,
+    )
+    cursor = db.conn.cursor()
+    trades = cursor.execute('SELECT * FROM trades WHERE id <> ? AND id <> ?', ("foo1", "foo2")).fetchall()  # noqa: E501
```

Comment:
please also in the beginning of the test assert that these trades are in the v31 DB with the mistaken fees.

Also don't use `<>` but `!=` as we do in the rest of the codebase.